hw/ip/edn/data/edn.hjson

```diff
@@ -0,0 +1,264 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "edn",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  param_list: [
+    { name: "NumEndPoints",
+      desc: "Number of end points to be supported by this block",
+      type: "int",
+      default: "4",
+      local: "true"
```

Comment:
I assume this has a maximum value of 4, given that there are only four `EndPointBusWidth*` entries below? Probably document this?

Reply:
Although a valid observation, the design constraints of this block are not locked down. Either this parameter should be removed and be locked down at 4, or it remains and the buses are all locked at the same fixed size. This is a limitation between verilog flexibility and the open titan interconnect scheme.

Reply:
Desc comment added!  Stay tuned for update.